Makefile

```diff
@@ -235,6 +235,14 @@ all: development standalone installer server
 all: revpdfprinter revandroid
 all: lc-run lc-test
 
+server-all: server
```

Comment:
Could you please add "Server Targets" section of the file for the server-specific targets?  Also, please make `all` require `server-all` instead of listing all the bits of the server stuff twice. :-)